app/assess/models/assessor_task_list.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class _SubCriteria:
+    id: str
+    name: str
+
+
+@dataclass
+class _Section:
+    name: str
+    sub_criterias: List[_SubCriteria]
+
+
+@dataclass
+class _CriteriaSubCriteria(_SubCriteria):
+    status: str
+    score: int
+    theme_count: int
+
+
+@dataclass
+class _Criteria:
+    name: str
+    total_criteria_score: int
+    total_criteria_score_possible: int
+    weighting: float
+    sub_criterias: List[_CriteriaSubCriteria]
+
+
+@dataclass
+class AssessorTaskList:
+    fund_name: str
+    project_name: str
+    project_reference: str
+    workflow_status: str
+    date_submitted: str
+    sections: List[_Section]
+    criterias: List[_Criteria]
+
+    @classmethod
+    def from_json(cls, json: dict):
+        return AssessorTaskList(
+            fund_name=json.get("fund_name"),
+            project_reference=json.get("project_reference"),
+            project_name=json.get("project_name"),
+            date_submitted=json.get("date_submitted"),
+            workflow_status=json.get("workflow_status"),
+            sections=[
+                _Section(
+                    name=section["name"],
+                    sub_criterias=[
+                        _SubCriteria(
+                            id=sub_criteria["id"],
+                            name=sub_criteria["name"],
+                        )
+                        for sub_criteria in section["sub_criterias"]
+                    ],
+                )
+                for section in json["sections"]
+            ],
+            criterias=[
+                _Criteria(
+                    name=criteria["name"],
+                    total_criteria_score=criteria["total_criteria_score"],
+                    total_criteria_score_possible=criteria[
+                        "total_criteria_score_possible"
+                    ],
+                    weighting=criteria["weighting"],
+                    sub_criterias=[
+                        _CriteriaSubCriteria(
+                            id=sub_criteria["id"],
+                            name=sub_criteria["name"],
+                            status=sub_criteria["status"],
```

Comment:
I think we are reading the mapping config here which doesnt contain any state. We wont be saving scores and statuses to the config dict, they should be in the assessment database. 
The statuses and scoring will come from the db records associated with the criteria/sub_critera. I think this Is linked to my comment on the assessment store PR, the state needs to be matched there before sending over to assessment.
so Assessment store reads (and matches) the config to the db records and then returns the sections and the ordered criteria with associated criteria score and sub_criteria statuses
Happy to chat this through on a call

Reply:
> I think we are reading the mapping config here which doesnt contain any state

The structure is similar to the mapping config, however there are subtle differences, the root level dataclass `AssessorTaskList` is meant to represent a model which has only the state necessary to be rendered on the UI.

State can be added as part of this transformation:  [assessment_mapping_fund_round.py#L30-L39](https://github.com/communitiesuk/funding-service-design-assessment-store/blob/cf75286e37b0cfe1f95423e8aea0ae793ec6502d/config/mappings/assessment_mapping_fund_round.py#L30-L39)

We will have to query the database for this state though.  

> the state needs to be matched there before sending over to assessment

This is what's happening in the above assessment_store function, however there aren't any database queries to get the state and then transform it, there are just placeholders at the moment -- I think I could build out the queries based on the current structure/data model & doing TDD, but I was reluctant to as they might have changes from the work Gio is doing at the moment on scores (also wanted to get feedback on approach so far before continuing).